src/borg/repository.py

```diff
@@ -685,8 +685,6 @@ def report_error(msg):
         return not error_found or repair
 
     def rollback(self, cleanup=False):
-        """
-        """
```

Comment:
another unrelated change.

suggestion: if you find such stuff, make a note, but do not change it, but stay focussed on the topic of the changeset you want to make.